src/osdc/Objecter.cc

```diff
@@ -2736,6 +2774,9 @@ int Objecter::_calc_target(op_target_t *t, bool any_change)
 	t->sort_bitwise,
 	sort_bitwise,
 	pg_t(prev_seed, pgid.pool(), pgid.preferred()))) {
+    // If the interval has changed, then don't resend for repair read
```

Comment:
won't this make the librados op hang? seems like we should resend, and let the osd give us the error if the interval changed